helm/values-uat.yaml

```diff
@@ -21,9 +21,14 @@ microservice-chart:
       REDIS_PORT: "6380"
       REDIS_SSL_ENABLED: "true"
       CARTS_MAX_ALLOWED_PAYMENT_NOTICES: "5"
+      ELASTIC_APM_SERVICE_NAME: pagopa-ecommerce-payment-requests-service
```

Comment:
I would name this service `pagopa-ecommerce-payment-requests-service-blue` so that we can distinguish blue instance logs from the green one, what do you think?

Reply:
Good idea 👍